synapse/tests/test_cortex.py

```diff
@@ -2471,6 +2470,37 @@ class Bawk:
             with self.raises(s_exc.NoSuchPivot):
                 nodes = await alist(core.eval('inet:ipv4 -> test:str'))
 
+    async def test_storm_expressions(self):
+        async with self.getTestCore() as core:
+
+            async def _test(q, ansr):
+                nodes = await core.eval(f'[test:int={q}]').list()
+                self.len(1, nodes)
+                self.eq(nodes[0].ndef, ('test:int', ansr))
+
+            await _test('$(42)', 42)
+            await _test('$(2 + 4)', 6)
+            await _test('$(4 - 2)', 2)
+            await _test('$(4 -2)', 2)
+            await _test('$(2 * 4)', 8)
+            await _test('$(1 + 2 * 4)', 9)
+            await _test('$(1 + 2 * 4)', 9)
+            await _test('$((1 + 2) * 4)', 12)
+            await _test('$(1 < 1)', 0)
+            await _test('$(1 <= 1)', 1)
+            await _test('$(1 > 1)', 0)
+            await _test('$(1 >= 1)', 1)
+            await _test('$(1 >= 1 + 1)', 0)
+            await _test('$(1 >= 1 + 1 * -2)', 1)
+            await _test('$(1 - 1 - 1)', -1)
+            await _test('$(4 / 2 / 2)', 1)
```

Comment:
what about 1/2?
what about 1/0?

Reply:
1/0 raises an exception. Not sure if that's the right thing to do or not.

Reply:
Deferring this. We need a generic way of moving along.